sdk/lib/atl/atltrace.h

```diff
@@ -0,0 +1,203 @@
+/*
+ * PROJECT:     ReactOS ATL
+ * LICENSE:     LGPL-2.0-or-later (https://spdx.org/licenses/LGPL-2.0-or-later)
+ * PURPOSE:     Providing ATLTRACE macro
+ * COPYRIGHT:   Copyright 2022 Katayama Hirofumi MZ <katayama.hirofumi.mz@gmail.com>
+ */
+ 
+#pragma once
+
+#include "atldef.h"
+
+#ifdef NDEBUG
+    #undef DBG
+    #undef _DEBUG
+#endif
+
+#if DBG && !defined(_DEBUG)
+    #define _DEBUG
+#endif
+
+#ifdef _DEBUG
+
+#include <stdio.h>
+
+namespace ATL
+{
+
+struct CTraceCategory
+{
+    explicit CTraceCategory(LPCSTR name, UINT level = 0) { }
+
+    operator UINT() const { return (1 << 15); }
```

Comment:
Shouldn't this be the category instead of a hardcoded number?

Reply:
OK. Fixed.